governance/SDK.md

```diff
@@ -0,0 +1,163 @@
+4. It supports the last N-1 major versions of CloudEvents spec, as defined in
+   [Contribution Acceptance](../SDK.md#contribution-acceptance)
+5. Issues and PRs are triaged (labeled, commented, reviewed, etc) regularly
+
+We define a project `cloudevents/sdk-x` _not actively maintained_ if:
+
+6. No commits on master from `sdk-x-maintainers` since 4 months
+7. Issues and/or PRs are not being triaged from `sdk-x-mantainers` since 2
+   months
+8. Security patches are not being **released** from `sdk-x-mantainers` since 1
+   months from CVE disclosures
+
+To prevent the project from becoming _not actively maintained_, the community MAY takes the following actions:
+
+- Temporary security patches delivered by an _sdk maintainer_ not part of
+  `sdk-x-maintainers` group
+- Handover of the project to a new maintainer
+- Archive the project
+
+### Security patches
+
+If a project `cloudevents/sdk-x` meets _healthy_ criteria 1, 2, 4, but the
+`sdk-x-maintainers` group is not actively performing security patches as defined
+in criteria 8, the community could decide to entitle one or more members from
+the larger _sdk maintainers_ group to perform security patches and release new
+minor versions of the project.
+
+Because the people entitled to perform security patches need write permissions on the repository, the community MUST temporarily grant these permissions.
+After the release of the patch, these permissions MUST be revoked.
+
+Because of the urgent nature of security patches, an _informal_ vote during the
+CloudEvents SDK meeting is enough to assign the security patch to the _sdk
+maintainer(s)_ who volunteers to take care of it.
+
+NOTE: this process is valid only for security patches, not for bugs or
+enhancements.
+
+### Handover to a new maintainer/group of maintainers
+
+If a project is not meeting the criteria 1, 2, 4, 5, 6, 7, 8, the community
+MAY decides to handover the project to a new maintainer/group of
+maintainers. The community can perform the handover to a new maintainer
+if all the following conditions are met:
+
+* A vote to add the new maintainer MUST have **already taken place** with
+  all people entitled to vote abstaining.
+* One of the following conditions had been met:
+  * The new maintainer(s) already meet the requirements enlisted in [New maintainers section](#new-maintainers)
+  * The new maintainer(s) performed the required PRs, Reviews and Code changes to meet the [New maintainers criteria](#new-maintainers) but they were unable to merge them due to inactivity of the current maintainer(s).
+
+The new maintainer must show the good will to keep the project healthy and
+community could ask for an eventual plan to improve the quality of the project.
+
+Once a new maintainer is identified, the community can proceed with a vote
+to handover the project using the
+[Asynchronous voting process](#asynchronous-voting-process). If more than one
+new maintainer wants to handover the project, separate voting process must be
```

Comment:
s/handover/maintain/

Reply:
I noticed that you have "must" here, not "MUST". I'd remove the uppercase version of the RFC2119 keywords in this doc since it's not a spec.